### DISCUSSION TITLE
Slidebars not being displayed correctly (Web-app in Python with Streamlit)

### DISCUSSION THREAD

**Discussion by mahnoor-byte** [AammarTufail/python_ka_chilla2023#102] [Q&A]
We should get different slidebars for each classifier. But I am getting only one slidebar for param "K " for all 3 classifiers. 

Here is the code of function defined for params for classifiers
![image](https://user-images.githubusercontent.com/121869407/216812500-bd89be13-24a0-49a4-9a38-008befbf595d.png)

Here is the out put.
![image](https://user-images.githubusercontent.com/121869407/216812425-a68e5fdc-8569-4393-80f0-f7e67276a534.png)

**Comment by AzamBukhari** [comment]
Please clarify the point below
1. When you select different datasets from the pull down menu e.g. IRIS. Wine etc.. Does your numbers change in the right hand side panel? Shape of dataset and number of classes?

**Comment by Muhammad-Ali-Butt** [comment]
send then entire code pic
